pages/home/internals/components/NoteForm.js

```diff
@@ -1,16 +1,37 @@
 export default function NoteForm(props) {
-  const { clickTextArea, inputTitleClass, buttonTitleClass, rows } = props;
+  const {
+    onSubmit,
+    clickAdd,
+    titleChange,
+    contentChange,
+    titleValue,
+    contentValue,
+    clickTextArea,
+    inputTitleClass,
+    buttonTitleClass,
+    rows,
+  } = props;
```

Comment:
**note:** If the `NoteForm` component is only used in one place, consider slimming down unnecessary props.